src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/LocalizedTimedTextTrackUpload/LanguageSelect/index.tsx

```diff
@@ -98,14 +105,19 @@ export const LanguageSelect = ({
   return (
     <Select
```

Comment:
Can the list be longer? Only 4 countries are display and the list is very long. In master branch 8 are display.

Reply:
As talked, the select is now searchable: [0449192](https://github.com/openfun/marsha/pull/2400/commits/04491922110a8b658b9960daa5be10bc27caa359)